### ISSUE TITLE
Colors not visible in certain terminals with a common color theme called "One Half Dark"

### ISSUE THREAD

**Issue by thernstig** [kimmobrunfeldt/concurrently#259]
I am running [Window Terminal](https://github.com/microsoft/terminal) with a very common theme called `One Half Dark` https://github.com/sonph/onehalf.

Here is some output from concurrently:

```sh
[2] npm --prefix server/ ci server/ exited with code 0
[1]
[1] > fsevents@1.2.7 install /home/etobher/code/cnom/eui_client/node_modules/fsevents
[1] > node install
```

This looks like this for me:
![colors](https://user-images.githubusercontent.com/30827238/107757609-12692e80-6d26-11eb-8061-2695501b55f6.png)

No other CLI tools I use have trouble with the color output in this theme, so I am thinking there is something iffy with concurrently and how it uses colors.

**Comment by milahu** [comment]
duplicate of https://github.com/kimmobrunfeldt/concurrently/issues/275

**Comment by gustavohenke** [comment]
Heya! Indeed, this has been fixed with v6.1.0.
